make/test.mk

```diff
@@ -75,6 +75,8 @@ test-olm-integration: push-operator-image olm-integration-setup
 	$(Q)sed -e "s,REPLACE_IMAGE,$(DEVCONSOLE_OPERATOR_REGISTRY_IMAGE):$(devconsole_version)-$(TAG)," ./test/e2e/catalog_source.yaml | oc apply -f -
 	$(Q)oc apply -f ./test/e2e/subscription.yaml
 
+	@-oc delete clusterroles.rbac.authorization.k8s.io "devconsole-operator"
```

Comment:
Use `$(Q)-` wrapper here and next line.
Also add a comment says that this required due to a potential bug in the test framework.